app/lib/server/server.go

```diff
@@ -319,10 +324,10 @@ func entityListToPublics(el *sda.Roster) []abstract.Point {
 	return publics
 }
 
-func isPublicIP(ip string) bool {
+func isPublicIP(ip network.Address) bool {
```

Comment:
It's already gone in `refactor_mocking_address` but the comments still sticks here I guess.